### PR TITLE
Hotfix: Success page layout broken due to duplicate "When"

### PR DESCRIPTION
**Pull request by hariombalhara**
**Before**:
Screenshot 2022-05-10 at 1 41 03 PM

**After**:
Screenshot 2022-05-10 at 1 41 30 PM

### CODE REVIEW

File: apps/web/pages/success.tsx [hunk 1]
```diff
@@ -299,15 +299,15 @@ export default function Success(props: SuccessProps) {
...
                           <div className="font-medium">{t("when")}</div>
-                          <div className="col-span-2 mb-6">
+                          {/* <div className="col-span-2 mb-6">
                             {date.format("MMMM DD, YYYY")}
                             <br />
                             {date.format("LT")} - {date.add(props.eventType.length, "m").format("LT")}{" "}
                             <span className="text-bookinglight">
                               ({localStorage.getItem("timeOption.preferredTimeZone") || dayjs.tz.guess()})
                             </span>
-                          </div>
```
Comment (alishaz-polymath) [inline apps/web/pages/success.tsx:309]: Why were we doing this in the first place? 🤔

**Review by alishaz-polymath — APPROVED** [review]

**Review by sean-brydon — APPROVED** [review]

File: apps/web/pages/success.tsx (hunk 1 above)
Comment (hariombalhara) [reply apps/web/pages/success.tsx:309]: Looks like a merge conflict resolution problem to me. Not sure. @leog @joeauyeung  Do you have any idea

**Comment by zomars** [comment]
@hariombalhara make sure to merge this to `main` as well.

**Comment by hariombalhara** [comment]
I can simply do a production branch merge to main?

**Comment by zomars** [comment]
That would mess up `main` git history. Let's keep them individual.

**Pull request merged**